cpp/README.md

```diff
@@ -242,7 +242,7 @@ The Unsubscribe functions clean up the socket and queues from a previous Subscri
 
 ```
 ZendarError NextImage(shannon::data::Image& proto, int timeout = -1);
-ZendarError NextTracker(zen::tracker::message::TrackerState& proto,
+ZendarError NextTracker(zpb::tracker::message::TrackerState& proto,
                 Int timeout = -1);
 ZendarError NextTracklog(shannon::data::Position& proto, int = -1);
 ZendarError NextLogMessage(shannon::data::LogRecord& proto, int = -1);
```

Comment:
I believe the `shannon::data` namespace is now the `zpb::data` namespace, so this comment should be updated.

Reply:
Cool. Let me take care of that.